### PR TITLE
[DOC] Refer to nilearn datasets module 

### LINKED ISSUE #4319: [DOC] Unclear what are nilearn datasets on the contributing page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe your proposed suggestion in detail.

Minor issue.

[This section](https://nilearn.github.io/stable/development.html#report-bugs-or-discuss-enhancement-ideas) encourages the use of Nilearn data while contributing, but it's unclear what is Nilearn data.

Should point to [nilearn.datasets](https://nilearn.github.io/stable/modules/datasets.html)

### List any pages that would be impacted.

`CONTRIBUTING.rst`

### PR DESCRIPTION
**Pull request by man-shu**
- Closes #4319

Changes proposed in this pull request:

- refer to `nilearn.datasets` [documentation](https://nilearn.github.io/stable/modules/datasets.html)

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ How to get help?
 
 If you have issues when using Nilearn, or if you have questions on how to use it, please don't hesitate to reach out!
 
-There are currently three ways to interact with the Nilearn team: through the :neurostars:`neurostars <>` forum, our :nilearn-gh:`github <>` issues, and through our weekly :nilearn-gh:`drop-in hours <issues/2882>`, usually **every Wednesday from 4pm to 5pm UTC**.
+There are currently three ways to interact with the Nilearn team: through the :neurostars:`neurostars <>` forum, our :nilearn-gh:`github <>` issues, and through our weekly `drop-in hours <https://tinyurl.com/nilearn-drop-in-hour>`_, usually **every Wednesday from 4pm to 5pm UTC**.
+We post on our `X account <https://twitter.com/nilearn>`_ in advance to let you know if the drop-in hours are happening that week.
```
Comment (Remi-Gau) [inline CONTRIBUTING.rst:15]: will rebase on the main branch of upstream to get read of this duplicate change from #4318

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ We welcome open discussion around improvements---both to the documentation as we
   to make sure it has not already been reported. If the bug has not been reported yet,
   create an :nilearn-gh:`new issue <issues/new/choose>`
   including a `minimal runnable example <https://stackoverflow.com/help/minimal-reproducible-example>`_
-  to showcase it (using Nilearn data) as well as your OS and Nilearn version.
+  to showcase it (using `Nilearn data <https://nilearn.github.io/stable/modules/datasets.html>`_) as well as your OS and Nilearn version.
```
Comment (Remi-Gau) [inline CONTRIBUTING.rst:37]: I think it is better to use the sphinx reference

https://github.com/nilearn/nilearn/blob/638648ba8b0e8f11cc67991d0d5a36c517cc095b/doc/modules/datasets.rst?plain=1#L1

and let sphinx handle the creation of internal links.

https://www.sphinx-doc.org/en/master/usage/referencing.html#cross-referencing-arbitrary-locations

Comment (Remi-Gau) [reply CONTRIBUTING.rst:37]: There is also an example on line 39 below

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
use internal sphinx reference rather than external links

**Review by Remi-Gau — APPROVED** [review]

**Review by bthirion — APPROVED** [review]
Thx !

**Pull request merged**